core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -95,12 +120,14 @@ object Controller {
 
     // akka-style factory to create a Controller object
     private class ServiceBuilder(config: WhiskConfig, instance: Int, logging: Logging) extends Creator[Controller] {
-        def create = new Controller(config, instance, logging)
+        // this method is not reached unless ExecManifest was initialized successfully
+        val runtimes = ExecManifest.runtimesManifest.getOrElse(Runtimes(Set()))
```

Comment:
there was an assert there which i lost in git surgery - you make a good point, might as well throw an exception in the getter for this.